posthog/test/test_middleware.py

```diff
@@ -123,7 +123,7 @@ class TestAutoProjectMiddleware(APIBaseTest):
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
-        cls.base_app_num_queries = 42
+        cls.base_app_num_queries = 43
```

Comment:
An extra PoE check when making the modifiers. Soon all PoE checks will be consolidated.